deploy/images/db/ebi.sql

```diff
@@ -1,85 +1,73 @@
 
--- Special view for EBI Data-Out
-CREATE VIEW local_ega.ebi_files AS
-SELECT id                                       AS file_id,
-       stable_id                                AS file_name,
-       archive_file_reference                     AS file_path,
-       archive_file_type                          AS file_type,
-       archive_file_size                          AS file_size,
-       archive_file_checksum                      AS unencrypted_checksum,
-       archive_file_checksum_type                 AS unencrypted_checksum_type,
-       header                                   AS header,
-       created_by                               AS created_by,
-       last_modified_by                         AS last_updated_by,
-       created_at                               AS created,
-       last_modified                            AS last_updated
+-- Special view for EBI Data-Out as per eu.elixir.ega.ebi.downloader.domain.entity.File
+CREATE VIEW local_ega_download.file AS
+SELECT stable_id                                AS file_id,
+       archive_file_reference                   AS file_name,
+       archive_file_reference                   AS file_path,
+       reverse(split_part(reverse(submission_file_path::text), '/'::text, 1)) AS display_file_name,
+       archive_file_size                        AS file_size,
+       NULL::text                               AS checksum,
+       NULL::text                               AS checksum_type,
+       archive_file_checksum                    AS unencrypted_checksum,
+       archive_file_checksum_type               AS unencrypted_checksum_type,
+       status                                   AS file_status,
+       header                                   AS header
 FROM local_ega.main
 WHERE status = 'READY';
 
-
--- Relation File <-> Index File
-CREATE TABLE local_ega.index_files (
+-- Relation File EGAF <-> Dataset EGAD
+CREATE TABLE local_ega.filedataset (
```

Comment:
Not found of having the filedataset table in the "main" schema, as I don't think that table should even be existing.

Why the need to the table here, and a view a little below in the local_ega_download schema?

Reply:
Ideally it could even be a different database, so I don't see the need to mix it here.

Reply:
Hold on a little bit longer, until Oscar brings the modifications we talked about yesterday.
I think it'll make the database structure clearer, and overall better.

Reply:
Moved that to it's own schema. The table has a reference to the id number while the data-out view needs the stable_id.